### PR TITLE
Mongo7 upgrade

### PR DESCRIPTION
**Pull request by Xiangs18**

### CODE REVIEW

**Review by MrCreosote — COMMENTED** [review]
LGTM, but @briehl should confirm

File: requirements.txt [hunk 1]
```diff
@@ -4,6 +4,6 @@ requests==2.20.0
 gunicorn==19.9.0
 gevent==1.3.7
 cachetools==2.1.0
-pymongo==3.7.2
+pymongo==4.7.2
```
Comment (briehl) [inline requirements.txt:7]: Anything preventing the latest one? 4.10.1?

**Review by briehl — COMMENTED** [review]
Just a pymongo version question.

File: requirements.txt (hunk 1 above)
Comment (Xiangs18) [reply requirements.txt:7]: No, I used version 4.7.2 in the `handle_service` back in May and am trying to make it consistent.
https://github.com/kbase/handle_service2/blob/069e04e6689eb1bd606ffe4018af3ff534924ac5/Pipfile#L9

Comment (Xiangs18) [reply requirements.txt:7]: I have tried the following commits to upgrade PyMongo to 4.10.1, but none of them worked. Converting to Pipenv might solve the problem, but that would require significant changes.

Comment (briehl) [reply requirements.txt:7]: Ok, that's fine. Just wondering.

**Review by briehl — APPROVED** [review]
👍

**Pull request merged**